src/backend/partaj/core/views/common.py

```diff
@@ -73,13 +74,13 @@ def get(self, request):
             writer.writerow(
                 [
                     str(ref.id),
-                    str(ref.object),
+                    ref.object,
                     ref.topic.name,
                     " - ".join([unit.name for unit in ref.units.all()]),
                     ref.due_date.strftime("%m/%d/%Y"),
                     " - ".join([user.get_full_name() for user in ref.users.all()]),
                     " - ".join([user.get_full_name() for user in ref.assignees.all()]),
-                    _(str(ref.state)),
+                    _(ref.state),
```

Comment:
`ReferralState` as a subclass of `TextChoices` already includes translations for the options.

```suggestion
                    models.ReferralState(ref.state).label,
```